### ISSUE TITLE
Logo should be output using image tag, not CSS

### ISSUE THREAD

**Issue by jdleonard** [Drupal-NYC/drupalcampnyc#101]
Assuming that the logo can be output effectively using an image tag (rather than as a background image), that is more accessible. See also #100
